eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/MutableMapIterable.java

```diff
@@ -172,6 +172,12 @@ default V getOrDefault(Object key, V defaultValue)
         return this;
     }
 
+    default MutableMapIterable<K, V> withMapIterable(MapIterable<? extends K, ? extends V> mapIterable)
```

Comment:
I dont think you need default method here because all the implementing interfaces should have a default method. It is easy to check by removing this method, or you can check the coverage and this method will be uncovered in this interface. If it is uncovered you can delete it.
That said, it is good to have default methods to help users with custom implementations, so I would say, you can add a test to cover this method. It can be done separately as a different PR.

Reply:
Thanks @nikhilnanivadekar for reviewing. If you don't mind, can I do this on another PR? Thanks again for your comments!

Reply:
Sure!